src/utils_GUI/GUI_utils.jl

```diff
@@ -398,17 +422,63 @@ function initialize_available_data!(gui)
 end
 
 """
-    get_JuMP_axes(var::JuMP.Containers.DenseAxisArray, i_T::Int)
+    get_investment_times(gui::GUI, max_inst::Float64)
 
-Get arrays of indices of a model variable `var` not being the time dimension (located at index `i_T`)
+Calculate when investments has taken place and store the information. An investement is
+assumed to have taken place if any `investment_indicators` are larger than get_var(gui,:tol)
+relative to `max_inst`.
 """
-function get_JuMP_axes(var::JuMP.Containers.DenseAxisArray, i_T::Int)
-    return axes(var)[vcat(1:(i_T - 1), (i_T + 1):end)]
+function get_investment_times(gui::GUI, max_inst::Float64)
+    T = gui.design.system[:T]
+    𝒯ᴵⁿᵛ = strategic_periods(T)
+    investment_indicators = get_var(gui, :descriptive_names)[:investment_indicators]
+    capex_fields = get_var(gui, :descriptive_names)[:total][:capex_fields]
+    period_labels = get_var(gui, :periods_labels)
+    model = get_model(gui)
+    for component ∈ get_root_design(gui)
+        element = get_element(component)
+        investment_times::Vector{String} = Vector{String}[]
+        investment_capex::Vector{Float64} = Vector{Float64}[]
+        for (i, t) ∈ enumerate(𝒯ᴵⁿᵛ)
```

Comment:
Both for loops could be single line, i.e., 
```suggestion
        for (i, t) ∈ enumerate(𝒯ᴵⁿᵛ), investment_indicator ∈ investment_indicators
```

Reply:
no, as explained in the comment in this line, the break statement would then break out of both loops in that case.

Reply:
True, did not consider that.